### PR TITLE
Migrate to webpack

### PR DESCRIPTION
**Pull request by gijskant**
- Update Angular to 1.7.8
- Update front-end dependencies
- Migrate front-end build system from gulp to webpack

To do:
- [ ] Fix karma tests
- [x] Run e2e tests
- [x] Test with MS Edge
- [x] Test with MS IE11

### CODE REVIEW

File: src/main/java/business/controllers/CustomErrorController.java [hunk 1]
```diff
@@ -1,46 +1,46 @@
...
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.Map;
+
+@RestController
+public class CustomErrorController implements ErrorController {
+
+    private static final String PATH = "/api/error";
```
Comment (gijskant) [inline src/main/java/business/controllers/CustomErrorController.java]: Should this not be `/error`?

File: webpack/webpack.common.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2017. The Hyve and respective contributors
```
Comment (ewelinagr) [inline webpack/webpack.common.js]: Please relicence to sth like:
/**
 * Copyright (C) 2019  Stichting PALGA
 * This file is distributed under the GNU Affero General Public License
 * (see accompanying file LICENSE).
 */

**Comment by ewelinagr** [comment]
Invalid logo image link here: https://github.com/thehyve/dntp-portal/blob/f6ff10f7992e10f6f3202753630b468dc8379427/src/main/webapp/app/lab-request/LabRequestController.js#L688

File: src/main/java/business/controllers/ProfileController.java [hunk 3]
```diff
@@ -1,100 +1,100 @@
...
+        // Query user's profile
+        log.info("GET labs profile for user with id " + user.getId());
+
+        ProfileRepresentation profileRepresentation = new ProfileRepresentation(userService.findOne(user.getId()));
+
+        // Return the representation
+        return labService.findAll(profileRepresentation.getHubLabIds());
+    }
+
+    @RequestMapping(value = "/profile", method = RequestMethod.PUT)
```
Comment (ewelinagr) [inline src/main/java/business/controllers/ProfileController.java]:
```suggestion
    @RequestMapping(value = "/api/profile", method = RequestMethod.PUT)
```

File: src/main/java/business/controllers/RequestController.java [hunk 4]
```diff
@@ -275,17 +275,17 @@ public RequestRepresentation submit(
     @PreAuthorize("isAuthenticated() and hasRole('palga')")
     @RequestMapping(value = "/requestNumbers/fix", method = RequestMethod.PUT)
```
Comment (ewelinagr) [inline src/main/java/business/controllers/RequestController.java]:
```suggestion
    @RequestMapping(value = "/api/requestNumbers/fix", method = RequestMethod.PUT)
```

File: src/main/webapp/app/forgot-password/ResetPasswordController.js [hunk 5]
```diff
@@ -46,9 +48,9 @@ angular.module('ProcessApp.controllers')
                     $scope.submitted = true;
 
                     // PUT to server (token and new password)
-                    Restangular.one('password').post('reset', { 
-                        token: $routeParams.token, 
-                        password: $scope.password 
+                    Restangular.one('password').post('reset', {
```
Comment (ewelinagr) [inline src/main/webapp/app/forgot-password/ResetPasswordController.js]:
```suggestion
                    Restangular.one('api/password').post('reset', {
```

File: src/main/webapp/app/forgot-password/ForgotPasswordController.js [hunk 6]
```diff
@@ -19,7 +22,7 @@ angular.module('ProcessApp.controllers')
                 $scope.error = '';
 
                 // PUT to server (old and new password)
-                Restangular.one('password/request-new').customPUT({ email: $scope.email }).then(function () {
+                Restangular.one('/api/password/request-new').customPUT({ email: $scope.email }).then(function () {
```
Comment (ewelinagr) [inline src/main/webapp/app/forgot-password/ForgotPasswordController.js]:
```suggestion
                Restangular.one('api/password/request-new').customPUT({ email: $scope.email }).then(function () {
```

File: src/main/webapp/app/activation/ActivationController.js [hunk 7]
```diff
@@ -3,21 +3,21 @@
...
+angular.module('ProcessApp.controllers')
+    .controller('ActivationController',['$scope', '$routeParams', 'Restangular',
+        function ($scope, $routeParams, Restangular) {
+            'use strict';
+
+            $scope.success = undefined;
+
+            // Check if the activation token is valid
+            $scope.init = function() {
+                Restangular.one('/api/register/users/activate/' + $routeParams.token).get().then(function () {
```
Comment (ewelinagr) [inline src/main/webapp/app/activation/ActivationController.js]:
```suggestion
                Restangular.one('api/register/users/activate/' + $routeParams.token).get().then(function () {
```

**Comment by gijskant** [comment]
All e2e tests are passing now.

**Comment by gijskant** [comment]
Currently fails on IE11: `Object doesn't support property or method 'includes'`

**Pull request merged**